### ISSUE TITLE
ImportValidator doesn't like relative imports

### ISSUE THREAD

**Issue by wsanchez** [DamnWidget/anaconda#432]
If I have an import such as:

``` python
from .scope import Scope
```

I get an error:

```
[E] ImportValidator (801): can't import .scope Scope
```

I suspect, but don't know how to confirm, that this may be related to how one invokes the python interpreter; for example:

``` console
wsanchez$ python deps/component.py
Traceback (most recent call last):
  File "deps/component.py", line 10, in <module>
    from .scope import Scope
ValueError: Attempted relative import in non-package
wsanchez$ python -m deps.component
wsanchez$ 
```

**Comment by DamnWidget** [comment]
Well that is because `scope.py` is not part of a package (as his parent has not be loaded) so it's normal that you get the import error as anaconda has no clue about how to import it into your current file.

**Comment by wsanchez** [comment]
Uh, well it *is* part of a package…  I have:

 - deps/
   - __init__.py
   - component.py
   - scope.py

The above import is in component.py.  This works in actual usage, but `ImportValidator` dislikes it.

**Comment by DamnWidget** [comment]
Sorry for the delay, being honest I totally forgot about this issue. I can't replicate it myself. Are you still experiencing the same behavior?

![screenshot_20160511_220332](https://cloud.githubusercontent.com/assets/329766/15196609/1933180e-17c5-11e6-87b4-3b826fb9bb86.png)

**Comment by QasimK** [comment]
I am experiencing this issue.

I have a Django project and I am using a virtualenv. It detects invalid imports from Django just fine. However, inside one of my own packages in my repository, it does not detect the relative import as being okay. Here is a picture (focusing on the problem):

![anabad](https://cloud.githubusercontent.com/assets/10064144/18888226/5297b0e0-84ef-11e6-87af-a49de9cbb63a.png)

**Comment by StanBright** [comment]
I've been "fighting" with this issue all day. Any updates on this? 

I'm using virtualenv, too. If that helps, I noticed that it sometimes starts/stops working after restart of Sublime O_o.

**Comment by FrnkVieira** [comment]
I'm also fighting with this issue. Anaconda does not handle relative imports properly when there are two files with the same name but in different directories being imported by relative imports.

**Comment by QasimK** [comment]
@DamnWidget Given the last few posts (especially @FrnkVieira's), could we drop the not replicable tag?

**Comment by DamnWidget** [comment]
This is probably related to #488 (possible a duplicate)

**Comment by DamnWidget** [comment]
I released a fix for #488 that could probably fix this issue as well, can you confirm please?

**Comment by DamnWidget** [comment]
@QasimK I will release a new version with specific fixes for this issue, I could not reproduce the behaviour expressed by @FrnkVieira in https://github.com/DamnWidget/anaconda/issues/432#issuecomment-260044701
So for resume, v2.1.10 will be fixed
